op-acceptance-tests/justfile

```diff
@@ -26,4 +68,58 @@ acceptance-test devnet="simple" gate="localnet":
         --testdir "/go/src/github.com/ethereum-optimism/optimism" \
         --gate {{gate}} \
         --validators /acceptance-tests.yaml \
-        --log.level info
+        --log.level debug
+
+# Helper to download the native binary from GitHub releases (internal use)
```

Comment:
just an idea, but what about using mise for this?
The ubi backend is pretty much doing exactly this, as far as I can tell

Reply:
I've changed it to be mise-managed. Neatens the justfile, for sure.
Only downside is the version of op-acceptor is now managed outside this dir (root mise.toml) but I think that's fine.
Thanks!

Reply:
This required deep changes to our CCI config, as downloading files from Github made us hit their default ratelimit, so I needed to incorporate the GITHUB_TOKEN into our jobs which require mise.